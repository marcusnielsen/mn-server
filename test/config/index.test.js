import Config from "../../src/config";

test("config", () => {
  const config = Config({ processEnv: process.env });
  expect(config).toMatchSnapshot();
});
