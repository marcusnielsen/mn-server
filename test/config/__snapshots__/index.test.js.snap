// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`config 1`] = `
Object {
  "nodeEnv": "test",
}
`;
